### DISCUSSION TITLE
An error when I build examples using cmake

### DISCUSSION THREAD

**Discussion by Monsterever** [CGAL/cgal#8287] [Q&A] [answered]
The information of this error is following, and it happens when I build with Eigen. 
The version of Eigen is 3.4.0, and CMake 3.18.6.
I find an answer about CMP0064 in #5859 in 2021, and it is fixed in CGAL-5.6.1. But it happens today.

```
CMake Warning (dev) at C:/local/CGAL-5.6.1/cmake/modules/CGAL_add_test.cmake:173 (if):
  Policy CMP0064 is not set: Support new TEST if() operator.  Run "cmake
  --help-policy CMP0064" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  TEST will be interpreted as an operator when the policy is set to NEW.
  Since the policy is not set the OLD behavior will be used.
Call Stack (most recent call first):
  C:/local/CGAL-5.6.1/cmake/modules/CGAL_add_test.cmake:354 (cgal_setup_test_properties)
  Mesh_3/CMakeLists.txt:126 (cgal_add_test)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Error at C:/local/CGAL-5.6.1/cmake/modules/CGAL_add_test.cmake:173 (if):
  if given arguments:

    "NOT" "TEST" "copy source_dir of Mesh_3_Examples"

  Unknown arguments specified
Call Stack (most recent call first):
  C:/local/CGAL-5.6.1/cmake/modules/CGAL_add_test.cmake:354 (cgal_setup_test_properties)
  Mesh_3/CMakeLists.txt:126 (cgal_add_test)
```

**Comment by lrineau** [comment]
What were your binary and source directories, during that failed CMake configuration? I see something odd in the call stack.

**Reply by Monsterever** [reply]
The source directiory is C:/Users/MSI/Desktop/algorithm/CGAL-5.6.1/examples
The binary directiory is C:/Users/MSI/Desktop/algorithm/CGAL-5.6.1/examples/build

**Reply by lrineau** [reply]
> The source directiory is C:/Users/MSI/Desktop/algorithm/CGAL-5.6.1/examples The binary directiory is C:/Users/MSI/Desktop/algorithm/CGAL-5.6.1/examples/build

I am sorry : I cannot reproduce the issue, even with CMake-3.18.6 like you.

Can you explain how to run cmake?

**Reply by Monsterever** [reply, marked as answer]
Oh, thank you for your help, it is my mistake. I know what happened, and I apologize for this due to my wrong change in CMakeLists. It works now.
